taxi/forms.py

```diff
@@ -0,0 +1,60 @@
+import re
+
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+from django.forms import ModelForm
+
+from taxi.models import Car, Driver
+
+
+class DriverLicenseUpdateForm(ModelForm):
+    LICENSE_REQUIRED_LENGTH = 8
+
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        pattern = r"^[A-Z]{3}[0-9]{5}"
+
+        if (len(license_number) != self.LICENSE_REQUIRED_LENGTH
+                or not re.match(pattern=pattern, string=license_number)):
+            raise ValidationError(
+                "License number invalid! Please enter correct license number."
+            )
+
```

Comment:
Maybe it is better to specify every particular error for better user expirience

Reply:
Sometimes it's even better not to specify, so malevolent person won't break in.
Consider how easy would be to understand each particular reason, why fake license isn't going inside. And how hard it gonna be to get right point, if you don't have that license and can't simply recheck what you typed.